src/FilterList.js

```diff
@@ -173,11 +173,33 @@ let FilterList = _.flow(
               style={bdJoin(child)}
             />
           ) : (
-            <div key={child.path} className="filter-list-item">
+            <div
+              key={child.path}
+              className="filter-list-item"
+              style={{
+                willChange: 'box-shadow',
+                transition: `box-shadow .4s ease-${
+                  child.paused ? 'in' : 'out'
+                }`,
+                boxShadow:
+                  child !== _.last(_.get('children', node)) && !child.paused
+                    ? 'rgba(0, 0, 0, 0.1) 0 -16px 20px -20px inset'
+                    : 'rgba(0, 0, 0, 0.1) 0 0 20px -20px inset',
+              }}
+            >
               <Label tree={tree} node={child} fields={fields}>
                 {mapNodeToLabel(child, fields)}
               </Label>
-              {!child.paused && (
+              <div
+                style={{
+                  maxHeight: child.paused ? 0 : '80vh',
+                  overflowY: child.paused ? 'hidden' : 'auto',
+                  willChange: 'max-height',
+                  transition: `max-height .3s ease-${
+                    child.paused ? 'out' : 'in'
+                  }`,
+                }}
+              >
```

Comment:
same as above - perhaps we can factor out an accordion component that can go on the theme, unified with an accordion we add to grey vest

Reply:
Moved to GreyVest theme